bd_spot_wrapper/spot_wrapper/spot.py

```diff
@@ -851,8 +851,9 @@ def get_camera_intrinsics(self, source, quality=None, pixel_format=None):
         return cam_intrinsics
 
     # MAYBE WE DONT NEED THIS IN SPOT.PY???????
+    @staticmethod
     def convert_transformation_from_sophus_to_magnum(
```

Comment:
I haven't worked with Magnus data-types before, can you add a link here as a comment for me to look? Thanks!

Reply:
You can find more info about magnum.Matrix4 [here](https://doc.magnum.graphics/python/magnum/Matrix4/)

magnum also supports other types too .. like Matrix3, etc.